public/js/Tree.js

```diff
@@ -929,9 +929,17 @@ Tree.prototype.centerOnNode = function(node)
 	{
 		var style = self.locals.style.get(this);
 
+		isResource = this.__data__.id.charAt(0) === 'r';
+
 		switch (style.level)
 			{
+
 				case -1:
+				case 0:
+					if(isResource){
+						self.nodeClicked(this);
+						break;
```

Comment:
This break placed here makes a bug. You only break from this case if the if statement is true. If it's false, then the case falls through to the next case, which does exactly the same thing that the if was gating. I don't think there's really a reason to do this check at all, it's okay if you can click on the center node. 

I also don't think it's necessary to center on resources to begin with. It just adds one more click when the user wants to go back